### PR TITLE
hardening/1288_add_note_about_ngsi_version

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #1288

### CODE REVIEW

File: doc/cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:5050/notify"
+    },
+    "attrs": [ "speed", "oil_level" ],
+    "attrsformat": "legacy"
+  },
+  "expires": "2020-12-31T00:00:00.000Z",
+  "throttling": 5
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md]: If the idea is to make this example "parallel" to the one for NGSIv1 subs, then throttling should be 1 instead of 5.

File: doc/cygnus-ngsi/installation_and_administration_guide/README.md [hunk 2]
```diff
@@ -12,12 +12,14 @@
...
     * [Running as a service](./running_as_service.md)
-* Advanced features:
+* Advanced topics:
     * [Testing](./testing.md)
     * [Name mappings](./name_mappings.md)
     * [Grouping Rules](./grouping_rules.md)
     * [Multitenancy](./multitenancy.md)
     * [Performance tips](./performance_tips.md)
+    * [NGSIv2 support](./ngsiv2_support.md)
+    * [IPv6 support](./ipv6_support.md)
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/README.md]: ipv6_support.md is not part of this PR... I understand this is just a fix in the RTD index and that .md was added in a previous PR, isn't it?

Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/README.md]: Yes, I noticed the entry was missing in the README.md, but the .md file exists from a long time ago.

File: doc/cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md (hunk 1 above)
Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1290/commits/3c9a517796f3979ee18b9001ae1ff502d000b5f9

File: doc/cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+#NGSIv2 support
+Currently, cygnus-ngsi does not support [NGSIv2](http://telefonicaid.github.io/fiware-orion/api/v2/stable/). Only notification format within NGSIv1 is accepted by `NGSIRestHandler`, i.e. something like:
```
Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1290/commits/3c9a517796f3979ee18b9001ae1ff502d000b5f9

**Comment by fgalan** [comment]
LGTM
